scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -192,6 +237,15 @@ namespace <%= namespace %> {
             );
         }
 
+        private void AvailableShippingRatesPoll(Action<List<string>, string> callback) {
+            PollTillReady(
+                checkout => {
+                    return checkout.availableShippingRates().ready();
+                },
+                callback
```

Comment:
Should you update `CurrentCheckout` before calling the `callback` otherwise `CurrentCheckout` will still have `availableShippingRates().ready() == false` even though it is good to go.